tests/t288_arg_oct.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import re
+
+from runtest import TestBase
+
+class TestCase(TestBase):
+    def __init__(self):
+        TestBase.__init__(self, 'arg_oct', result="""
+# DURATION     TID     FUNCTION
+   0.560 us [1587104] | __monstartup();
+   0.170 us [1587104] | __cxa_atexit();
+            [1587104] | main() {
+ 120.272 us [1587104] |   creat("bar.foo", 0755) = 4;
+   3.210 us [1587104] |   chmod("bar.foo", 0777) = 0;
+   6.350 us [1587104] |   unlink("bar.foo") = 0;
+ 131.752 us [1587104] | } = 0; /* main */
+""", cflags='-g')
+
+    def build(self, name, cflags='', ldflags=''):
+        if not "dwarf" in self.feature:
+            return TestBase.TEST_SKIP
+        # cygprof doesn't support arguments now
+        if cflags.find('-finstrument-functions') >= 0:
+            return TestBase.TEST_SKIP
```

Comment:
Function arguments for library calls can be captured even for `-finstrument-functions` compiled binaries. So this is not needed as well.